explorer/app/hooks/useEntityData.tsx

```diff
@@ -61,7 +63,7 @@ export const useEntityListData = (
  * @returns an object with the loaded data and a flag indicating is the data is loading
  */
 export const useEntityDetailData = (
```

Comment:
Can you rename this to `useFetchEntity`, and then also move `useEntityListData` to its own file called  `useFetchEntities`?  I think I would like to follow a single-hook-per-hook-file pattern. Let's see how it goes!